bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -21,10 +21,23 @@ pub struct ExecutionResult {
     create_address: Option<ffi::evmc_address>,
 }
 
+/// EVMC execution message structure.
+pub struct ExecutionMessage {
+    pub kind: ffi::evmc_call_kind,
```

Comment:
We could consider replacing pubs with getters and it that case we could just take a reference to `ffi::evmc_message`.

Reply:
That makes sense. the pub stuff belongs only in FFI IMO